cli/command/trust/signer_remove_test.go

```diff
@@ -71,9 +71,9 @@ func TestTrustSignerRemoveErrors(t *testing.T) {
 func TestRemoveSingleSigner(t *testing.T) {
 	cli := test.NewFakeCli(&fakeClient{})
 	cli.SetNotaryClient(notaryfake.GetLoadedNotaryRepository)
-	err := removeSingleSigner(cli, "signed-repo", "test", true)
+	_, err := removeSingleSigner(cli, "signed-repo", "test", true)
```

Comment:
Don't we want to check the returned boolean?

Reply:
will fix.